plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogMaterializedView.java

```diff
@@ -38,7 +38,7 @@
 import static io.trino.plugin.hive.metastore.glue.AwsSdkUtil.getPaginatedResults;
 import static io.trino.testing.TestingNames.randomNameSuffix;
 
-public class TestIcebergGlueCatalogMaterializedViewTest
+public class TestIcebergGlueCatalogMaterializedView
```

Comment:
Could you rename the class in `trino-iceberg/pom.xml` too?

Reply:
good catch, thanks!